pkg/process/checks/net.go

```diff
@@ -208,6 +209,14 @@ func batchConnections(
 			EncodedDNS:        dnsEncoder.Encode(batchDNS),
 			ContainerHostType: cfg.ContainerHostType,
 		}
+
+		// Add OS telemetry
+		if hostInfo := host.GetStatusInformation(); hostInfo != nil {
+			cc.KernelVersion = hostInfo.KernelVersion
+			cc.Architecture = hostInfo.KernelArch
+			cc.Platform = hostInfo.Platform
```

Comment:
I don't see a way around it, but we should be aware that this platform value may be unreliable when within a container. I looked at the library we use, and it relies a lot on files existing in `/etc` to determine the platform. That will differ from the kernel version, obtained via `uname`, which is passed through from the host/node kernel.

Reply:
Actually the underlying library parsing the `/etc` files (`gopsutil`) allows you to set `HOST_ETC`: https://github.com/shirou/gopsutil/blob/7e5409b1310ad25ab30db1bc270534f3776a5851/internal/common/common.go#L339-L341
So once we create the new mounting points and update our helm charts etc, this should work.

Reply:
... you can still have a scenario in which someone is running the new agent with an old helm chart though, so it's something to keep in mind